securitas/tests/unit/conftest.py

```diff
@@ -83,6 +83,26 @@ def dummy_user(make_user):
     yield
 
 
+@pytest.fixture
+def dummy_group():
+    ipa_admin.group_add('dummy-group', description="A dummy group")
+    yield
+    ipa_admin.group_del('dummy-group')
+
+
+@pytest.fixture
+def dummy_user_as_group_manager(logged_in_dummy_user, dummy_group):
+    """Make the dummy user a manager of the dummy-group group."""
+    ipa_admin.group_add_member("dummy-group", users="dummy")
+    ipa_admin.group_add_member_manager("dummy-group", users="dummy")
+    yield
+
+
+@pytest.fixture
+def remove_password_min_time():
+    ipa_admin.pwpolicy_mod("dummy-group", 0)
```

Comment:
I think it'd be clearer and more future-proof if you used the keyword here for the second argument (`krbminpwdlife=0`).
Also, I think it would be better to restore the value on teardown (after a `yield`) so that it does not affect other tests running after the test that has used this fixture. It can look kinda useless since now we don't see why we would ever need the krbminpwlife to be more than zero, but I think it's a good practice for the testsuite to leave the directory in the state that we found it in.

Reply:
Yeah, the reason I didn't reset it was because the group gets deleted by dummy_group anyway. I will add it though just for completeness.

Reply:
Oh, I thought it removed the limit for everyone, but indeed it's only for the dummy-group... Then you should depend on the `dummy_group` fixture, and yeah it's not as necessary to remove it afterwards. Up to you :-)

Reply:
OK now I get it, you actually want to add a password policy for the dummy-group where the minlifetime is zero. The command you want to use then is `pwpolicy_add`, not `pwpolicy_mod`.
I suggest the following changes:
```patch
--- a/securitas/security/ipa.py
+++ b/securitas/security/ipa.py
@@ -46,7 +46,7 @@ class Client(IPAClient):
             parse_group_management_error(data)
         return data['result']
 
-    def pwpolicy_mod(
+    def pwpolicy_add(
         self, group, krbminpwdlife=None, **kwargs
     ):
         """
@@ -57,7 +57,7 @@ class Client(IPAClient):
         """
         params = {'all': True, 'raw': True, 'krbminpwdlife': krbminpwdlife}
         params.update(kwargs)
-        data = self._request('pwpolicy_mod', group, params)
+        data = self._request('pwpolicy_add', group, params)
         return data['result']
 
 
--- a/securitas/security/ipa_admin.py
+++ b/securitas/security/ipa_admin.py
@@ -13,7 +13,7 @@ class IPAAdmin(object):
         "group_del",
         "group_add_member",
         "group_add_member_manager",
-        "pwpolicy_mod"
+        "pwpolicy_add",
     )
 
     def __init__(self, app):
--- a/securitas/tests/unit/conftest.py
+++ b/securitas/tests/unit/conftest.py
@@ -100,7 +100,7 @@ def dummy_user_as_group_manager(logged_in_dummy_user, dummy_group):
 
 @pytest.fixture
 def remove_password_min_time():
-    ipa_admin.pwpolicy_mod("dummy-group", 0)
+    ipa_admin.pwpolicy_add("dummy-group", krbminpwdlife=0, cospriority=10)
 
 
 @pytest.fixture
```